docs/src/pages/patterns/loading.js

```diff
@@ -0,0 +1,340 @@
+} from "../../components";
+import loadingImage from "../../images/patterns/loading/loading.svg";
+import skeletonScreenImage from "../../images/patterns/loading/skeleton-screen.png";
+import loadingInlineImage from "../../images/patterns/loading/loading-inline.svg";
+
+export default () => (
+  <Layout>
+    <Helmet>
+      <title>Loading Content</title>
+    </Helmet>
+    <Intro>
+      <Title>Loading Content</Title>
+      <IntroText>
+        Loading patterns are used when a user’s action causes a delay in the
+        system’s response. A successful loading indicator provides timely
+        feedback on the progress of the user’s request and creates an illusion
+        of time going by faster.
+      </IntroText>
+      <Box width="240px" mt="96px" mb="x6" mx="auto">
+        <img src={loadingImage} alt="Loading animation" />
+      </Box>
+    </Intro>
+
+    <DocSection>
+      <Text>
+        Nulogy Design System provides a range of loading methods to match
+        various situations.
+      </Text>
+      <SectionTitle>Skeleton screens (progressive loading)</SectionTitle>
+      <Text mb="x5">
+        The skeleton screens deliver information in increments as they are
+        becoming available, making the loading experience fluid and seamless.
+        The skeleton screens are used for page or section loads and typically
+        complete within 5 seconds, even though the duration of 10 seconds is
+        acceptable. For delays over 10 seconds, other methods that provide more
+        feedback should be considered.
+      </Text>
+      <Box
+        width={{ medium: "110%", large: "150%" }}
+        pt="x3"
+        ml={{ medium: "-5%", large: "-25%" }}
+      >
+        <img
+          src={skeletonScreenImage}
+          alt="Loading progression of skeleton screens"
+        />
+      </Box>
+      <Text fontSize="small" color="darkGrey" textAlign="center" mt="x2">
+        3 stages of loading progression of skeleton screens
+      </Text>
+    </DocSection>
+
+    <DocSection>
+      <SectionTitle>Full-screen loading</SectionTitle>
+      <Text>
+        The full-screen method is used when an action temporarily disables the
+        application for more than 2 seconds. There are 2 types of full-screen
+        loaders: <strong>Indeterminate</strong> and <strong>Determinate</strong>
+        .
+      </Text>
+      <Text>
+        The indeterminate loading method is used for actions that can't be
+        estimated, unlike determinate loading which is used for actions that can
+        be estimated.
+      </Text>
+      <Text>
+        For delays between 2 and 5 seconds the animation is sufficient, while
+        for delays over 5 seconds the animation should be accompanied with the
+        appropriate combination of information and actions. If the action takes
+        longer than expected to execute, a warning with options to retry or
+        abort should be displayed. Examples of different variations are coming
```

Comment:
maybe the coming soon notice should be an alert?

Reply:
I added the alert.